common/supplemental/supplementalData.xml

```diff
@@ -1918,7 +1918,7 @@ XXX Code for transations where no currency is involved
 		<language type="mai" scripts="Tirh" territories="IN NP" alt="secondary"/>
 		<language type="mak" scripts="Latn"/>
 		<language type="mak" scripts="Bugi" territories="ID" alt="secondary"/>
-		<language type="man" scripts="Latn Nkoo"/>
+		<language type="man" scripts="Nkoo Latn"/>
```

Comment:
This is driven by the Ghana language entry saying there is a large Manding population using Nkoo in Ghana. Really `man` is a macrolanguage code so we shouldn't give that much weight to it. https://en.wikipedia.org/wiki/Manding_languages

Reply:
Can you check that again? Nkoo figures can be skewed by advocacy.

Reply:
I tried to find better figures but I could not verify Nko usage by Manding -- so I'll edit the population data so Latn takes precedence.